com.avaloq.tools.ddk.xtext.format.ui/src-gen/com/avaloq/tools/ddk/xtext/format/ide/contentassist/antlr/PartialFormatContentAssistParser.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * generated by Xtext 2.25.0
+ */
+package com.avaloq.tools.ddk.xtext.format.ide.contentassist.antlr;

```

Comment:
I have noticed now that this commit introduces '.ide' package. We need to move that to its own plug-in, so that it can be used on LSP without pulling the full UI one, but we can do this on another commit.